LanguageFeatures/Augmentation-libraries/augmenting_types_A10_t01.dart

```diff
@@ -8,8 +8,8 @@
 /// augmentation can add new members to an existing type.
 ///
 /// @description Checks that instance members defined in the body of an augment
-/// of a class, mixin, extension, or enum are added to an instance namespace of
-/// the corresponding type in the augmented library
+/// of a class, mixin, extension, enum or an extension type are added to an
+/// instance namespace of the corresponding type in the augmented library.
```

Comment:
We could have a notion of 'the instance namespace of a class' etc., but I think it's better to use 'interface', which is used in this manner many times in specification documents already:
```suggestion
/// of a class, mixin, extension, enum or an extension type are added to the
/// interface of the corresponding type in the augmented library.
```

Similarly for other locations where the phrase 'instance namespace' is used.

Reply:
Updated

Reply:
Thanks!